softhddev.c

```diff
@@ -2959,6 +2960,7 @@ int SetPlayMode(int play_mode)
 #ifdef DEBUG
 		    VideoSwitch = GetMsTicks();
 		    Debug(3, "video: new stream start\n");
+		    AudioStarted = 0;
```

Comment:
this should definitely be outside the #ifdef DEBUG

Reply:
Yes. My mistake.